src/MetaLoader.php

```diff
@@ -398,19 +398,20 @@ private function createContext(array $source): array
         return ['http' => ['header' => $rawheader]];
     }
 
-
-    /**
-     * @param array $source
-     */
     private function addCachedMetadata(array $source): void
     {
-        if (isset($this->oldMetadataSrc)) {
-            foreach ($this->types as $type) {
-                foreach ($this->oldMetadataSrc->getMetadataSet($type) as $entity) {
-                    if (array_key_exists('metarefresh:src', $entity)) {
-                        if ($entity['metarefresh:src'] == $source['src']) {
-                            $this->addMetadata($source['src'], $entity, $type);
-                        }
+        if (!isset($this->oldMetadataSrc)) {
+            Logger::info('No oldMetadataSrc, cannot re-use cached metadata');
+            return;
+        }
+
+        foreach ($this->types as $type) {
+            foreach ($this->oldMetadataSrc->getMetadataSet($type) as $entity) {
+                if (array_key_exists('metarefresh:src', $entity)) {
+                    if ($entity['metarefresh:src'] == $source['src']) {
+                        Logger::debug(sprintf('Adding cached meatadata for %s, %s',
```

Comment:
The codesniffer is going to complain this line.. Every parameter has to go on a new line.

Reply:
🤷‍♂️ See Actions, "opening parenthesis must be the last character in a multi-line call"